content/tutorials/getting-started.ja.markdown

```diff
@@ -0,0 +1,101 @@
+---
+タイトル: はじめに
+カテゴリ: 全般
```

Comment:
Rregarding category, I think it would be best to create a "General - Japanese" category right below the "General" section", so all the Japanese articles can be grouped together.

Please look at the `config.toml` file in the root folder, and copy the `params.tutorial_categories.general` section and call it `params.tutorial_categories.general_jp` with id `general_jp` with order 1.

Reply:
@KawaneRio Hello! Do you have time to look at the requested comments sometime this month? Would love to merge this.